### ISSUE TITLE
VolumeScreen semantics are confusing

### ISSUE THREAD

**Issue by fstanis** [google/horologist#689]
"47%, slider, swipe up or swipe down to adjust" is the current label for the `VolumeScreen` when using TalkBack. It seems this is the label that's automatically added by `Stepper`, but it can be confusing since swiping doesn't function for this screen.

We should look into overriding this somehow. The easiest option, if possible, is to remove the screen-level label and keep just  the one for volume buttons and device button.

**Comment by yschimke** [comment]
On Wear Compose 1.0, you can't override it without messing up a lot more.  https://issuetracker.google.com/issues/244409133

But can possible merge/clearAndSetSemantics?

Also a question of whether it should do announcements on volume/device changes? or live region?

**Comment by fstanis** [comment]
> Also a question of whether it should do announcements on volume/device changes? or live region?

Good point, let's check this with UX before we proceed.

**Comment by yschimke** [comment]
We should be a bit sticky about retaining the defaults for Wear Compose composables.  If we genuinely have a difference, then we do what we need.  But is this something about Stepper that should be changed generally?

**Comment by yschimke** [comment]
Closing, as I think this has been looked at. Reopen if still needed.
